test/files/embed-cockpit.html

```diff
@@ -7,9 +7,9 @@
     <link rel="shortcut icon" href="../dist/img/favicon.ico">
     <link href="../dist/css/patternfly.min.css" rel="stylesheet" media="screen, print">
     <link href="../dist/css/patternfly-additions.min.css" rel="stylesheet" media="screen, print">
-    <script src="../components/jquery/dist/jquery.min.js"></script>
-    <script src="../components/bootstrap/dist/js/bootstrap.min.js"></script>
-    <script src="../components/matchHeight/jquery.matchHeight-min.js"></script>
+    <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.matchHeight/0.6.0/jquery.matchHeight-min.js"></script>
```

Comment:
I'm fine with loading from the network temporarily, if that's what we did before.

However typically we want our integration tests to run without being dependent on data loaded over the network. I tried to make this happen with docker images, but I'm pretty sure we're not all the way there yet.